website/docs/r/spot_fleet_request.html.markdown

```diff
@@ -27,13 +27,15 @@ resource "aws_spot_fleet_request" "cheap_compute" {
     ami               = "ami-1234"
     spot_price        = "2.793"
     placement_tenancy = "dedicated"
+    iam_instance_profile_arn = "arn:aws:iam::12345678:instance-profile/webserver-role"
```

Comment:
We should either fix this so its a properly formatted ARN (account ID is 12 digits `123456789012`) or just reference a "fake" resource (`"${aws_iam_instance_profile.example.arn}"`). Also as a nitpick, if these are going to be added in here, please line up all the equals symbols similar to how `terraform fmt` would e.g.

```
ami                      = "ami-1234"
spot_price               = "2.793"
placement_tenancy        = "dedicated"
iam_instance_profile_arn = "arn:aws:iam::12345678:instance-profile/webserver-role"
```

Reply:
Done 👍